scout/server/blueprints/variant/templates/variant/buttons.html

```diff
@@ -34,8 +34,13 @@
 {% endmacro %}
 
 {% macro splice_junctions_button(institute_id, case_name, variant_id, omics_variant_id) %}
-  <a class="btn btn-sm btn-secondary text-white" href="{{url_for('alignviewers.sashimi_igv', institute_id=institute_id, case_name=case_name, variant_id=variant_id, omics_variant_id=omics_variant_id)}}" target="_blank"
+  {% if omics_variant_id %}
+    <a class="btn btn-sm btn-secondary text-white" href="{{url_for('alignviewers.sashimi_igv', institute_id=institute_id, case_name=case_name, variant_id=variant_id, omics_variant_id=omics_variant_id)}}" target="_blank"
```

Comment:
Do you have to pass also variant_id here?

Reply:
Pretty sure not having it messed with routing during initial tests, but we can try now!